.github/workflows/cicd-3-deploy.yaml

```diff
@@ -1,13 +1,43 @@
 name: "CI/CD deploy"
+run-name: CI/CD Deploy ${{ inputs.deployEnvironment }} << ${{ github.ref_name }}
 
 on:
   workflow_dispatch:
     inputs:
+      deployEnvironment:
+        description: Name of the environment to deploy
+        required: true
+        default: dev
+        type: string
+      terraformAction:
+        description: Terraform Action
+        required: true
+        default: apply
+        type: string
       tag:
         description: "This is the tag that is oging to be deployed"
         required: true
         default: "latest"
 
+env:
+  PROJECT_DIR: infrastructure
+  PROJECT_NAME: notify-web-platform
+  DEFAULT_ENVIRONMENT: dev
+  CI_PIPELINE_IID: ${{ github.run_number }}
+  CI_COMMIT_REF_NAME: ${{ github.ref_name }}
+  CI_COMMIT_BRANCH: ${{ github.ref_name }}
+  AWS_REGION: eu-west-2
+  DEPLOY_ENVIRONMENT: ${{ inputs.deployEnvironment }}
+  TERRAFORM_ACTION: ${{ inputs.terraformAction }}
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read  # This is required for actions/checkout
+
+concurrency:
+  group: caas-${{ github.workflow }}-${{ github.event.inputs.deployEnvironment }}
```

Comment:
I wouldn't expect to see the word "caas" anywhere.

Reply:
sorry typo